### PR TITLE
Br 04

### PR DESCRIPTION
**Pull request by MaliukDaria**

### CODE REVIEW

File: src/main/java/com/example/demo/controller/ProductController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/products")
+public class ProductController {
+    private static final String NUMBER_OF_PRODUCTS = "1000";
```
Comment (Sofasmile) [inline src/main/java/com/example/demo/controller/ProductController.java]:
```suggestion
    private static final String DEFAULT_NUMBER_OF_PRODUCTS = "1000";
```

File: src/main/java/com/example/demo/controller/ReviewController.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/reviews")
+public class ReviewController {
+    private static final String NUMBER_OF_WORDS = "1000";
```
Comment (Sofasmile) [inline src/main/java/com/example/demo/controller/ReviewController.java]:
```suggestion
    private static final String DEFAULT_NUMBER_OF_WORDS = "1000";
```
the same with user

File: src/main/java/com/example/demo/service/impl/ProductServiceImpl.java [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+import com.example.demo.model.dto.ProductResponseDto;
 import com.example.demo.repository.ProductRepository;
 import com.example.demo.service.ProductService;
 import java.util.List;
+import org.springframework.data.domain.PageRequest;
 import org.springframework.stereotype.Service;
 
 @Service
 public class ProductServiceImpl implements ProductService {
+    private static final int NUMBER_OF_MOST_COMMENTED_PRODUCTS = 10;
```
Comment (Sofasmile) [inline src/main/java/com/example/demo/service/impl/ProductServiceImpl.java]: do we need this constant?

**Review by Sofasmile — COMMENTED** [review]
great job

File: src/main/java/com/example/demo/service/impl/ProductServiceImpl.java (hunk 3 above)
Comment (MaliukDaria) [reply src/main/java/com/example/demo/service/impl/ProductServiceImpl.java]: no, I forgot to remove)

File: src/main/java/com/example/demo/controller/ProductController.java (hunk 1 above)
Comment (MaliukDaria) [reply src/main/java/com/example/demo/controller/ProductController.java]: fixed

File: src/main/java/com/example/demo/controller/ReviewController.java (hunk 2 above)
Comment (MaliukDaria) [reply src/main/java/com/example/demo/controller/ReviewController.java]: fixed

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**